### ISSUE TITLE
[pro] OperationStore HMAC failure

### ISSUE THREAD

**Issue by orionmcc** [rmosolgo/graphql-ruby#2270]
Hi. I've hit a new error while attempting to implement the  OperationStore in our code base (Kickstarter). I'm getting an HMAC failure while attempting to sync a sample file.

I installed graphql-ruby-client via yarn.

I created a new client on my local machine called Kickstarter with the secret c59af5ceacbc79e1a2cce5d20edb767938c383fead5ee836b592c6af1f3c222b

I created a test .graphql file with a simple query in it.

I run the command
```
 yarn run graphql-ruby-client sync --url="http://ksr.test/graphql/sync" --client=Kickstarter --secret="c59af5ceacbc79e1a2cce5d20edb767938c383fead5ee836b592c6af1f3c222b" --path="./test-graphql/*.graphql"
```

I get the response 
```
Sync failed:
  Server responded with 401
error An unexpected error occurred: "Command failed.
```

And when I check the Ruby logs I see
```
OperationStore: Failed to authorize body, header: "GraphQL::Pro Kickstarter cee96ed2c37832285ac5ae05c6ed23f23ad786508e272c649e50fccb19e7ea51" (Invalid HMAC digest)
```

Any thoughts on what's going on here? I don't think there's anything in our setup that would be causing the authentication to fail, but I'm open to idea. Thanks.

--Orion

**Comment by rmosolgo** [comment]
Hi! Sorry about the hangup on this one. I have two suggestions to debug: 

- I just released v1.6.4 of the javascript client which includes the new `--verbose` option. Could you update and re-run with `--verbose`, then share the result? 
- Also, I just shipped graphql-pro 1.10.5 which includes the _expected_ digest in the Rails log. Could you share the message that appears in the log? 
- `--verbose` _won't_ include the content of the files it's sending. Could you share the content of any files listed in the `--verbose` output? 

This is all supposed to Just Work :tm:, so this information will help us track down where it's falling apart!

**Comment by rmosolgo** [comment]
Happy to debug further if there's anything else to share! Feel free to reopen if you want to.
